src/Platform/index.js

```diff
@@ -109,12 +114,32 @@ export default {
       return getOrSet('network', params)
     },
   },
-  Accesibility: {
+  Accessibility: {
     closedCaptions(params) {
       return getOrSet('closedCaptions', params)
     },
     voiceGuidance(params) {
       return getOrSet('voiceGuidance', params)
     },
   },
+  Personalization: {
+    setEntitlements(entitlements) {
+      return getOrSet('entitlements', entitlements)
+    },
+    addToWatchHistory(history) {
+      return getOrSet('watchHistory', history)
+    }
+  },
+  Advertising: {
+    AdSkipTiers: adSkipTiersEnum,
```

Comment:
See comment above, it seems to me the code for a developer becomes more verbose with this approach. What use case does it solve in practice?

Reply:
Code completion, and self documenting code.